rotkehlchen/api/rest.py

```diff
@@ -4416,3 +4424,28 @@ def search_for_names_everywhere(self, addresses: List[ChecksumEthAddress]) -> Re
             addresses=addresses,
         )
         return api_response(_wrap_in_ok_result(mappings))
+
+    def get_config_arguments(self) -> Response:
+        max_size_in_mb_all_logs = self.rotkehlchen.args.max_size_in_mb_all_logs
```

Comment:
nitpick: Why add different vars for all of these? Just use them directly in the dict.

Reply:
Because is easier to read them since I'm using in the dict for the variable and for the `is_modified` value (it doesn't have all the dots for where to find them) and also makes lines shorter

Reply:
As I said in discord, I would have done it as I comment here since the lines would be much shorter and don't mind the dots for access, but dont' mind it.